### ISSUE TITLE
```Coq: Doc: Getting Started``` command.

### ISSUE THREAD

**Issue by rtetley** [coq-community/vscoq#862]
Following this [topic](https://coq.zulipchat.com/#narrow/stream/237662-VsCoq-devs-.26-users/topic/.E2.9C.94.20Goal.20panel.20not.20visible.20after.20updating.20it.20to.202.2E1.2E7) it feels clear that it is not always obvious for newer how to get started.

One reason in this case is the change in a default setting, which will be addressed in a separate issue.

The reason is we cruelly lack pointers towards how to get started in the Coq world.

We should address this by creating a splash screen which is always re-openable (in the vein of the Lean 4 Getting started page) to give instructions to new users.

**Comment by rtetley** [comment]
This will be done with a "walkthrough" contribution. Following inspiration from the lean page I think we should have: 
- [ ] First and foremost, an indication on how to get back to this page if need be
- [ ] Helpful resources (books and tutorials)
- [ ] How to install the required dependencies (first with a step by step guide, and then with an automatic command)
- [ ] How to set up a coq Project (not sure about this one) ?
- [ ] FAQ with a link to the zulip chat

**Comment by TheoWinterhalter** [comment]
For setting up a Coq project, unless VSCoq provides a snippet to do it automatically (like Lean), I'm not sure it's worth doing? I mean I guess not too much effort should be spent on this when this will be redundant with—hopefully—the Coq/Rocq website?

But maybe it could contain a link to the relevant tutorials / getting started / etc.

**Comment by rtetley** [comment]
I mean I don't think it would be too much effort creating a snippet that creates a new project. But can any of you help me with how that would work ? Just ask for a project name and then it creates a dir with _CoqProject and/or a Makefile as well as an empty coq file ? With some basic imports ? Idk, what is the workflow when you create a project ?

**Comment by patrick-nicodemus** [comment]
@TheoWinterhalter not sure how relevant this is to the overall thread but here is my experience with dune.
First, I switch between VSCoq and ProofGeneral.
I only know how to configure ProofGeneral using a _CoqProject file and unless someone shows me another way, I will probably continue to leave a _CoqProject in my file alongside a dune config file - the dune config file for building the project and the _CoqProject existing for the benefit of ProofGeneral. If they get out of sync it may be painful.

A basic dune project is explained reasonably well on the dune website. Mine looks like

- ./dune-project
```
(lang dune 3.16)
(using coq 0.8)
```
Note that 0.8 refers to the version of the Coq sublanguage of the dune configuration language.

- ./dune
```
(coq.theory
 (name MyTheory)
 (modules ;; I think this is optional if you just want to include all modules.
   ("A"
    "core"
    "nat_example"
    "eq_example"
    )
   )
 (coqdoc_flags ("-utf8"))
 (stdlib no) ;; The standard library isn't loaded.
 (theories Coq) ;; The namespace for the standard library Coq is accessible/linked, but not loaded.
)
```

**Comment by TheoWinterhalter** [comment]
I thought you could generate the `_CoqProject` from the `dune` file?

But what I meant is more: why should I switch to `dune`? It's a more lispy way of giving the same info. I've used for OCaml, so maybe it makes sense if your project is geared for extraction or builds some OCaml at some point.

If it's relevant, I guess you could have an option in the `Init project` command.

**Comment by rtetley** [comment]
Closed by #868 
I will open a separate issue for "setting up a coq project"
